errors.go

```diff
@@ -0,0 +1,5 @@
+package ringpop
+
+import "errors"
+
+var ErrNotBootstrapped = errors.New("Ringpop has not been bootstrapped")
```

Comment:
Errors in golang are usually lowercased. eg: http://blog.golang.org/error-handling-and-go